spec/mysql_helper.rb

```diff
@@ -28,7 +28,7 @@ def reset
   def dump
     buf = StringIO.new
     ActiveRecord::SchemaDumper.dump(ActiveRecord::Base.connection, buf)
-    buf = buf.string.sub(/\A.*\bActiveRecord::Schema\.define\(version: \d+\) do/m, '').sub(/end\s*\z/, '')
+    buf = buf.string.sub(/\A.*\bActiveRecord::Schema(?:\[[\d.]+?\])?\.define\(version: \d+\) do/m, '').sub(/end\s*\z/, '')
```

Comment:
Can we write this regular expression strict and simpler?

- Replace `.` (any character) with `\.` (dot itself)
- Remove the first`?` because the second `?` matches the case of `[7.0]` missing

```suggestion
    buf = buf.string.sub(/\A.*\bActiveRecord::Schema(?:\[[\d\.]+\])?\.define\(version: \d+\) do/m, '').sub(/end\s*\z/, '')
```

Reply:
> - Replace `.` (any character) with `\.` (dot itself)

Escaping the dot is redundant, and this will match only the dot as expected.

> - Remove the first`?` because the second `?` matches the case of `[7.0]` missing

Added a commit to remove it.